lib/types/types_test.go

```diff
@@ -90,7 +90,7 @@ func TestNullDecoder(t *testing.T) {
 	}
 
 	for k, v := range input {
-		t.Run("Error Message/"+k, func(t *testing.T) {
```

Comment:
This seems to have nothing to do with this PR ?

Reply:
Yes, seems Goland sucks when doing rename.

Reply:
Fixed